### PR TITLE
DOCSP-35148: Convert docs to Snooty RST

### PR DESCRIPTION
**Pull request by ccho-mongodb**
Convert Markdown documentation for Snooty RST to prepare for builds.

[JIRA](https://jira.mongodb.org/browse/DOCSP-35148)

### Checklist

- [x] Add tests and ensure they pass
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Update documentation for new features

### CODE REVIEW

**Comment by GromNaN** [comment]
You can add a note to the changelog saying the doc is converted to rst and hosted on the mongodb website.

Also, update the links in the readme to target the mongodb website.

**Comment by ccho-mongodb** [comment]
> You can add a note to the changelog saying the doc is converted to rst and hosted on the mongodb website.
> 
> Also, update the links in the readme to target the mongodb website.

The docs are not yet hosted on the MongoDB website since we're waiting on the submodule builds and breadcrumb link updates. I'll create a ticket to follow up with these changes when the docs are published.

**Review by GromNaN — APPROVED** [review]
SGTM

File: docs/eloquent-models.txt [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+.. meta::
+   :keywords: php framework, odm, code example
+
+This package includes a MongoDB enabled Eloquent class that you can use to
+define models for corresponding collections.
+
+Extending the base model
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get started, create a new model class in your ``app\Models\\`` directory.
```
Comment (GromNaN) [inline docs/eloquent-models.txt]: This double backslash is displayed: https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/#extending-the-base-model

Comment (ccho-mongodb) [reply docs/eloquent-models.txt]: > This double backslash is displayed: https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/#extending-the-base-model

Thanks for catching that, and sorry I missed it.
I will revert this and the one in Upgrading and double-check on staging before updating this PR.

Comment (ccho-mongodb) [reply docs/eloquent-models.txt]: Updated on staging:
Screenshot 2024-01-10 at 1 22 29 PM
Screenshot 2024-01-10 at 1 21 40 PM

**Comment by GromNaN** [comment]
Thank you Chris

**Pull request merged**